test/e2e/pods.go

```diff
@@ -351,6 +352,37 @@ var _ = Describe("Pods", func() {
 			Failf("Failed to delete pod: %v", err)
 		}
 
+		By("verifying the kubelet observed the termination notice")
+		pod, err = podClient.Get(pod.Name)
+		Expect(wait.Poll(time.Second*1, time.Second*30, func() (bool, error) {
```

Comment:
This is polling every second for up to 30 seconds, but the Client.Get() below has a timeout of one minute. Is that on purpose?